packages/mds-provider/api.ts

```diff
@@ -221,66 +204,56 @@ function api(app: express.Express): express.Express {
     return undefined
```

Comment:
We don't need to return anything here I think

Reply:
Actually, I think `return {}` is probably more appropriate.